server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketUserEndpoints.java

```diff
@@ -183,4 +183,23 @@ public ApiResponse delete(final ApiClient client, @ApiParameter("password") fina
 
         return new ApiResponse(ApiResponseType.OK);
     }
+
+    @ApiEndpoint("get_user")
+    public ApiResponse getUser(final ApiClient client, @ApiParameter("uuid") final UUID uuid) {
+        Session session = client.get(Session.class);
+        if (session == null || !session.isValid()) {
+            return new ApiResponse(ApiResponseType.FORBIDDEN, "NOT_LOGGED_IN");
+        }
+
+        session = Session.getById(uuid);
```

Comment:
It's about getting the serialized user information by its id not about getting a user by a session.